lib/gstreamer/d3d11/decoder.py

```diff
@@ -26,6 +26,13 @@ class DecoderTest(BaseDecoderTest):
   def before(self):
     super().before()
 
+  def get_supported_format_map(self):
+    return {
+      "I420"  : "yuv420p",
+      "NV12"  : "nv12",
+      "P010"  : "p010le",
+    }
+
```

Comment:
The gstreamer base decoder class has not been converted to use the BaseFormatMapper, yet.  Only the ffmpeg base decoder class uses it.
Thus, this member function will never be called.

Reply:
@FocusLuo can you reassign or close this.